hack/update-gofmt.sh

```diff
@@ -36,6 +36,7 @@ find_files() {
         -o -wholename './release' \
         -o -wholename './target' \
         -o -wholename '*/third_party/*' \
+        -o -wholename '*/_vendor/*' \
```

Comment:
Cool - I think skipping staging is a better answer, too.

On Thu, Sep 1, 2016 at 3:19 PM, Chao Xu <notifications@github.com> wrote:

> In hack/update-gofmt.sh
> <https://github.com/kubernetes/kubernetes/pull/31882#discussion_r77264322>
> :
>
> > @@ -36,6 +36,7 @@ find_files() {
> >          -o -wholename './release' \
> >          -o -wholename './target' \
> >          -o -wholename '*/third_party/*' \
> > +        -o -wholename '*/_vendor/*' \
>
> We can skip all of staging.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/31882/files/ddf8dab680e87b2ccb04b68fafbfd9dd11785ac7#r77264322>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVGdoS9iA51IpZ1VmAW5e5vTYcCfxks5ql09agaJpZM4JywL1>
> .
>